make/content/advanced.tex

```diff
@@ -44,16 +45,74 @@
     \end{minted}
   \end{center}
 
-  Lösung: manuell synchronisieren
+  Lösung: Ein \texttt{make}-Feature (\texttt{v4.3}): \emph{grouped targets}
   \begin{center}
-    $\vdots$ \\
+    \small
     \begin{minted}{make}
-      plot1.pdf: plot.py data.txt
+      all: report.txt
+
+      report.txt: plot1.pdf plot2.pdf
+        touch report.txt
```

Comment:
Die Indentierung hier ist incosistent. Make braucht Tabs, keine spaces.

Vermutlich besser alle Beispiele als echte Makefiles, z.B. in `examples/slideX.make` zu speichern und inputminted zu behmen